samples/highcharts/series-boxplot/whisker-length/demo.js

```diff
@@ -0,0 +1,51 @@
+Highcharts.chart('container', {
+    chart: {
+        type: 'boxplot'
+    },
+
+    title: {
+        text: 'Configuring whisker lengths'
+    },
+
+    yAxis: {
+        title: false
+    },
+
+    legend: {
+        enabled: false
+    },
+
+    series: [{
+        whiskerLength: 100,
+        lowerWhiskerLength: 10,
```

Comment:
Should we use percentages instead for all the values, here and below? Works better in a responsive scenario.